examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -38,10 +40,21 @@ var _ = ginkgo.BeforeSuite(func() {
 
 	randomEd25519AuthFactory := auth.NewED25519Factory(randomEd25519Priv)
 
+	vmConfig := vm.NewConfig()
+	vmConfig.Config = make(map[string]any)
+	vmConfig.Config["storeTransactions"] = true
+	vmConfig.Config["ws"] = ws.WSConfig{
+		MaxPendingMessages: 10_000,
+	}
+
+	configBytes, err := json.Marshal(vmConfig)
+	require.NoError(err)
+
```

Comment:
This should be in `integration.go` instead of coming from MorpheusVM

Reply:
done